### PR TITLE
[MSqlServerServer] | Release Note v1.0

### PR DESCRIPTION
**Pull request by DavoudEshtehari**

### CODE REVIEW

File: release-notes/MSqlServerServer/1.0/1.0.0.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Release Notes
+
+## General Availability of Microsoft.SqlServer.Server
+
+_**1.0.0 released 28 January 2022**_
+
+This is the initial public stable release of the **Microsoft.SqlServer.Server** namespace in a separate assembly. This library is a helper for **Microsoft.Data.SqlClient** enabling cross framework support of UDT types.
+
```
Comment (Kaur-Parminder) [inline release-notes/MSqlServerServer/1.0/1.0.0.md:8]: PR link is missing.

Comment (DavoudEshtehari) [reply release-notes/MSqlServerServer/1.0/1.0.0.md:8]: This is the very beginning release. Following similar releases, we didn't include any PR!

**Review by ErikEJ — COMMENTED** [review]
Does this enable any customer scenarios? If so, a sample would be great.

**Review by JRahnama — APPROVED** [review]

**Comment by David-Engel** [comment]
> Does this enable any customer scenarios? If so, a sample would be great.

Not directly/Not quite yet. This is one prerequisite in a series of steps for unblocking the team that is working on bringing spatial types to .NET Core/.NET.

**Review by Kaur-Parminder — APPROVED** [review]

**Comment by ErikEJ** [comment]
Excellent news! Thanks  for that.

**Review by johnnypham — APPROVED** [review]

File: release-notes/MSqlServerServer/1.0/1.0.0.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# Release Notes
+
+## General Availability of Microsoft.SqlServer.Server
+
+_**1.0.0 released 28 January 2022**_
+
+This is the initial public stable release of the **Microsoft.SqlServer.Server** namespace in a separate assembly. This library is a helper for **Microsoft.Data.SqlClient** enabling cross framework support of UDT types.
```
Comment (DavoudEshtehari) [inline release-notes/MSqlServerServer/1.0/1.0.0.md:7]:
```suggestion
This is the initial public stable release of the **Microsoft.SqlServer.Server** namespace in a separate assembly. This library is a dependency for **Microsoft.Data.SqlClient** enabling cross framework support of UDT types.
```

**Pull request merged**